libs/ebpfnetsh/tokens.h

```diff
@@ -17,7 +17,8 @@
 typedef enum
 {
     VL_NORMAL = 0,
+    VL_INFORMATIONAL = 2,
```

Comment:
suggest: It looks a bit odd to me to see ordering of 0, 2, 1 enum values. Should we change this to be in numerical order? Alternatively, is there any side effect of changing the enum value of informational to be 1 and verbose to be 2?

(Not a blocking comment)